toolkit/motors/rev_motors.py

```diff
@@ -87,6 +92,7 @@ def init(self):
         self.pid_controller = self.motor.getPIDController()
         self.encoder = self.motor.getEncoder()
         self._set_config(self._config)
+        self.is_init = True
```

Comment:
Please take this out. The is_init thing was a hack that we were using to get the tests to work. I don't think we need it now.